backend/services/bookingService.ts

```diff
@@ -109,19 +110,22 @@ export = class BookingService {
 
     const court = await this.locationService.getVideoLinkEnabledCourt(context, courtId)
 
-    // Do not await the result of this call: Fire and forget.
-    this.notificationService.sendBookingCreationEmails(context, currentUsername, {
-      agencyId,
-      court: court.name,
-      prison: agencyDetails.description,
-      offenderNo,
-      prisonerName: formatName(prisonBooking.firstName, prisonBooking.lastName),
-      date: mainStartTime,
-      preDetails: preAppointment?.description,
-      mainDetails: mainAppointment.description,
-      postDetails: postAppointment?.description,
-      comment,
-    })
+    try {
+      await this.notificationService.sendBookingCreationEmails(context, currentUsername, {
+        agencyId,
+        court: court.name,
+        prison: agencyDetails.description,
+        offenderNo,
+        prisonerName: formatName(prisonBooking.firstName, prisonBooking.lastName),
+        date: mainStartTime,
+        preDetails: preAppointment?.description,
+        mainDetails: mainAppointment.description,
+        postDetails: postAppointment?.description,
+        comment,
+      })
+    } catch (e) {
+      log.error('Notify failed: ', e)
+    }
```

Comment:
we can't just have hanging promises - we could `catch` these instead but this also works

(does potentially mean that it could hang on slow requests I guess..)